### PR TITLE
Add rotating buff for gemm_multi_d

### PR DESCRIPTION
**Pull request by zjing14**

### CODE REVIEW

File: cmake/gtest.cmake [hunk 1]
```diff
@@ -8,7 +8,8 @@ endif()
 
 FetchContent_Declare(
     GTest
-    GIT_REPOSITORY https://github.com/google/googletest.git
```
Comment (tenpercent) [inline cmake/gtest.cmake]: wonder if you need to commit this line?

Comment (zjing14) [reply cmake/gtest.cmake]: Fixed

**Comment by aska-0096** [comment]
LGTM. please merge after CI passed.
@junliume

**Review by carlushuang — APPROVED** [review]
LGTM

**Pull request merged**